### PR TITLE
Update windowsdefenderapplicationguard-csp.md

### PR DESCRIPTION
**Pull request by sazankha**

### CODE REVIEW

File: windows/client-management/mdm/windowsdefenderapplicationguard-csp.md [hunk 1]
```diff
@@ -275,6 +275,8 @@ Value type is integer. Supported operation is Get.
 - Bit 2	- Set to 1 when	the client machine has a valid OS license and SKU.
 - Bit 3	- Set to 1 when	Application Guard installed on the client machine.
 - Bit 4	- Set to 1 when	required Network Isolation Policies are configured.
+ > [!IMPORTANT]
+> If you are deploying Application Guard via Intune, Network Isolation Policy must to configured, to enable Application Guard for Microsoft Edge.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/windowsdefenderapplicationguard-csp.md:279]:
```suggestion
 > If you are deploying Application Guard via Intune, Network Isolation Policy must to configured, to enable Application Guard for Microsoft Edge. 
```

File: windows/client-management/mdm/windowsdefenderapplicationguard-csp.md [hunk 2]
```diff
@@ -275,6 +275,8 @@ Value type is integer. Supported operation is Get.
 - Bit 2	- Set to 1 when	the client machine has a valid OS license and SKU.
 - Bit 3	- Set to 1 when	Application Guard installed on the client machine.
 - Bit 4	- Set to 1 when	required Network Isolation Policies are configured.
+ > [!IMPORTANT]
+> If you are deploying Application Guard via Intune, Network Isolation Policy must to configured, to enable Application Guard for Microsoft Edge. 
 - Bit 5	- Set to 1 when the client machine meets minimum hardware requirements.
 - Bit 6 - Set to 1 when system reboot is required.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/windowsdefenderapplicationguard-csp.md:281]:
```suggestion
- Bit 6	- Set to 1 when system reboot is required.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sazankha, kindly commit the minor cosmetic suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

File: windows/client-management/mdm/windowsdefenderapplicationguard-csp.md [hunk 3]
```diff
@@ -275,8 +275,10 @@ Value type is integer. Supported operation is Get.
 - Bit 2	- Set to 1 when	the client machine has a valid OS license and SKU.
 - Bit 3	- Set to 1 when	Application Guard installed on the client machine.
 - Bit 4	- Set to 1 when	required Network Isolation Policies are configured.
+ > [!IMPORTANT]
+ > If you are deploying Application Guard via Intune, Network Isolation Policy must to configured, to enable Application Guard for Microsoft Edge.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/windowsdefenderapplicationguard-csp.md:279]:
```suggestion
 > If you are deploying Application Guard via Intune, Network Isolation Policy must be configured to enable Application Guard for Microsoft Edge. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @sazankha, one more please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @sazankha

**Pull request merged**